lib/kitchen/transport/winrm.rb

```diff
@@ -87,6 +87,10 @@ def close
         def execute(command)
           return if command.nil?
           logger.debug("[WinRM] #{self} (#{command})")
+
+          if command.length > MAX_COMMAND_SIZE
+            command = run_from_file_command(command)
```

Comment:
I'm leaning to stay with this as for this PR as it may have some unforseen side effects if applied globally and I'd want to add tests and do a bunch of validation.